lib/classes/class-bootstrap.php

```diff
@@ -1710,6 +1753,9 @@ public function wp_delete_site($old_site) {
 
         $sql = "DROP TABLE IF EXISTS $table_name";
         $wpdb->query($sql);
+
+        ud_stateless_db()->clear_db();
```

Comment:
@balexey88 let's use 1 function to remove all custom tables

Reply:
Done!
https://github.com/udx/wp-stateless/commit/e93b7550742f3e31acc6e30449340e30b70588b0